Example/YoshiExample/AppDelegate.swift

```diff
@@ -30,30 +30,55 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func setupDebugMenu() {
-        // YoshiMenuType.TableView
-        let menuItemProd = MenuItem(name: "Production", selected: true)
-        let menuItemStaging = MenuItem(name: "Staging")
-        let menuItemQA = MenuItem(name: "QA")
-        let environmentItems: [YoshiTableViewMenuItem] = [menuItemProd, menuItemStaging, menuItemQA]
-
-        let tableViewMenu = TableViewMenu(title: "Environment",
-            subtitle: nil,
-            displayItems: environmentItems, didSelectDisplayItem: { (displayItem) in
-            NSNotificationCenter.defaultCenter()
-                .postNotificationName(Notifications.EnvironmentUpdatedNotification, object: displayItem.name)
+        var menu = [YoshiMenu]()
+
+        menu.append(setupEnvironments())
+        menu.append(setupDateSelector())
+        menu.append(setupInstabug())
```

Comment:
i think it would read better if these `setup...` functions were named `environmentMenu()`, `dateSelectorMenu()`, `instabugMenu()` instead.

- [ ] update function names